ui/client/components/graph/ExpressionSuggester.js

```diff
@@ -2,9 +2,11 @@ import _ from 'lodash';
 
 export default class ExpressionSuggester {
 
-  constructor(typesInformation, variables) {
+  constructor(typesInformation, variables, processingType, httpService) {
     this._typesInformation = typesInformation
```

Comment:
Mayby it's a good time to change this underscore (private/protected) code style? :)

Reply:
What's wrong with it? This kind of refactoring I'll rather put to separate PR and fix it globally

Reply:
It's old and ugly pattern for simulation accessibility in JS.. Okey, I leave decision to you.